Unscheduled Care/2. Unscheduled Care outputs.R

```diff
@@ -308,7 +308,8 @@ emergency_adm_age <- emergency_adm %>%
   summarise(adm = sum(admissions)) %>%
   ungroup() %>%
   left_join(loc_pop_age1) %>%
-  mutate(data = round_half_up(adm / pop * 100000))
+  mutate(data = round_half_up(adm / pop * 100000)) %>%
+  filter(!is.na(year))
```

Comment:
`drop_na()` is from tidyr and is a slightly nicer way to achieve the same result. 
```suggestion
  drop_na(year)
```